index.md

```diff
@@ -4,14 +4,10 @@ root: .  # Is the only page that doesn't follow the pattern /:path/index.html
 permalink: index.html  # Is the only page that doesn't follow the pattern /:path/index.html
 ---
 
-This lesson is intended for people who would like to know more about working in a team in an agile manner.
+This lesson is intended for people that create research software
+who would like to learn more about working in a team.
 
-This lesson has the following episodes:
-
-- The advantages and disadvantages of working in a team
-- What working in an agile manner is and why it is a good way of organizing a team.
-- The two major agile methodologies: Scrum and Kanban
-- How agile can be applied in a (e-)scientific setting
+FIXME: Expand description
```

Comment:
Is there an issue for addressing this `FIXME`?

Reply:
good point, there is now: #31